styles/facts.css

```diff
@@ -0,0 +1,74 @@
+.container {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-gap: 10px;
+    max-width: 960;
+    margin: 0 auto;
+
+}
+body {
+    background-color: rgb(102, 192, 171);
+    } 
```

Comment:
This curly brace needs to be unindented so it's at the same level as the 'b' in body.